pkg/cmd/roachtest/tests/django_blocklist.go

```diff
@@ -168,9 +168,16 @@ var djangoBlocklists = blocklistsForVersion{
 }
 
 // Maintain that this list is alphabetized.
-var djangoBlocklist21_2 = blocklist{}
+var djangoBlocklist21_2 = blocklist{
```

Comment:
I think maybe we should put these in the "ignore list" instead since they are flaky. i don't believe they fail reliably.

also instead of "unknown" can it say "will be fixed in django-cockroachdb v3.2.2" (based on https://github.com/cockroachdb/cockroach/issues/65691#issuecomment-857203122)